src/libraries/pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -36,7 +38,9 @@
     <PrereleaseLibraryPackage Include="System.Runtime.Caching" />
     <PrereleaseLibraryPackage Include="System.Security.AccessControl" />
     <PrereleaseLibraryPackage Include="System.Security.Cryptography.Cng" />
-    <PrereleaseLibraryPackage Include="System.Security.Cryptography.Pkcs" />
+    <PrereleaseLibraryPackage Include="System.Security.Cryptography.Pkcs">
+      <Version>5.0.1</Version>
```

Comment:
@Anipik this is a case where I think our simpler baselining model could help.  We could leave everything at RTM here, and just record the servicing versions of packages as their new baseline.

Reply:
Yeah, exactly. The problem with the Compat pack is that it relies on `package-index` data as it uses `IndexedDependency` item.